apps/web/src/components/workflow/FlowEditor.tsx

```diff
@@ -219,33 +133,159 @@ export function FlowEditor({
 
       const parentNode = reactFlowInstance?.getNode(parentId);
 
-      if (typeof parentNode === 'undefined') {
+      if (typeof parentNode === 'undefined' || dropId !== '2') {
```

Comment:
What `dropId` with value 2 represents?

Reply:
Ok, I see the comment on the old code:
`dropId === 2 condition will active only DropZone and will inactive other nodes`

Could we store it with a meaningful name as a const for reference? Would like to do a suggestion but not familiar with the functionality.

Reply:
I missed it, I think we can remove it because I removed at the moment the new functionality of adding new nodes onDrop because it was not intuitive.